ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -310,12 +310,9 @@
     - is_primary
     - cloud_platform_is_aws or cloud_platform_is_gcp
 
-- name: Set op_defaults timeout [aws/gcp]
+- name: Set op_defaults timeout
   ansible.builtin.command:
-    cmd: >-
-      crm configure rsc_defaults
-      $id="op-options"
-      timeout=600
+    cmd: crm configure op_defaults timeout=600
```

Comment:
is this back compatible with tools provided with SLES < 15.6 ?

Reply:
Now that is completely removed, won't we need it somewhere?